airbyte-integrations/connectors/source-harvest/source_harvest/availability_strategy.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+
+import logging
+from typing import Dict
+
+import requests
+from airbyte_cdk.sources import Source
+from airbyte_cdk.sources.streams import Stream
+from airbyte_cdk.sources.streams.http.availability_strategy import HttpAvailabilityStrategy
+from requests import HTTPError
+
+
+class HarvestAvailabilityStrategy(HttpAvailabilityStrategy):
+    """
+    This class is tested as part of test_source.check_connection
+    """
+
+    def reasons_for_unavailable_status_codes(
+        self, stream: Stream, logger: logging.Logger, source: Source, error: HTTPError
+    ) -> Dict[int, str]:
+        reasons_for_codes: Dict[int, str] = {
+            requests.codes.UNAUTHORIZED: "Please ensure your credentials as valid.",
```

Comment:
I tried to commit @ChristoGrab 's suggestion but it didn't commit. I'll update it manual. Thanks for the suggestion Christo ❤️

Reply:
Haha yes sorry, first time using the `suggested change` feature and I was worried about messing up the indentation so I switched it to a manual review 😅